### ISSUE TITLE
Remove gas check in giveSorted

### ISSUE THREAD

**Issue by adi44** [razor-network/contracts#177]
Givesorted function makes use gasleft() function. need to check how much gas is consumed?

**Comment by adi44** [comment]
According hardhat gas reporter it consumes 184169

**Comment by SkandaBhat** [comment]
We need to calculate the gas consumed for these 2 lines:

```
        disputes[epoch][msg.sender].lastVisited = lastVisited;
        disputes[epoch][msg.sender].accWeight = accWeight;
```

https://github.com/razor-network/contracts/blob/master/contracts/Core/BlockManager.sol#L213

**Comment by hrishikeshio** [comment]
solved in #263
